locale/en/user.po

```diff
@@ -852,3 +852,18 @@ msgstr "This ORCID has not been verified. Please remove this unverified ORCID an
 
 msgid "user.removeRole.message"
 msgstr "Are you sure want remove this role permanently?"
+
+msgid "user.futureRole.notification.message"
+msgstr "Your role is scheduled to begin on "
```

Comment:
I wonder if the scheduled begin date should be a variable in the locale key it is for others directly below. It would make it clear what goes here as well as allow for other languages where the date would not appear at the end to insert it as appropriate.